.github/pull_request_template.rst

```diff
@@ -0,0 +1,15 @@
+Summary
```

Comment:
why this changes are coming in this PR? @oakbani

Reply:
Updated base branch with master